### ISSUE TITLE
Pooling Rage

### ISSUE THREAD

**Issue by Jinnai** [timhul/ClassicSim#116]
As a warrior you often start encounters with pooled rage. It would be nice to have some possibility to control how much rage the simulation starts with.

**Comment by timhul** [comment]
Yes, this should be made available. It is essentially a part of a larger project I've wanted to do for a long time, which is an extensible way of implementing class settings so that it is easy to implement arbitrary settings like this one for any class.
